apps/backoffice/src/components/services/service-status.tsx

```diff
@@ -13,15 +13,14 @@ export type ServiceStatusProps = {
 export const ServiceStatus = ({ status }: ServiceStatusProps) => {
   const { t } = useTranslation();
   const [color, setColor] = useState<string>("default");
+  const [disabled, setDisabled] = useState(false);
 
   useEffect(() => {
     switch (status?.value) {
       case ServiceLifecycleStatusTypeEnum.approved:
         setColor("success");
         break;
       case ServiceLifecycleStatusTypeEnum.deleted:
```

Comment:
is this case still needed ?

Reply:
@StefanoDoc Yes, because `draft` and `deleted` are used to set _default_ color, and then also, `deleted` is used to set "disabled" behaviour to _MUI Chip_ component, as requested by our quality assurance issue.

Reply:
if you mean depending on the fact that, in any case, the _default_ of the switch is equal to the `draft` value and `deleted` ok, I could also delete the two cases.
But, perhaps, this way it is easier to understand which color is assigned for each state.